client/packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -374,7 +374,7 @@ export const getOutboundQueries = (sdk: Sdk, storeId: string) => ({
           ({ type, numberOfPacks, isUpdated, isCreated }) =>
             type === InvoiceLineNodeType.UnallocatedStock &&
             numberOfPacks === 0 &&
-            isUpdated &&
+            !isUpdated &&
```

Comment:
can you just remove the `isUpdated` check please. If a line has a placeholder quantity, and then stock comes in and the line is allocated, then the placeholder quantity could be reduced to `0` and then the `isUpdated` flag is set to true. We'd want those placeholders removed too!

Reply:
Ah yes. I'd been wondering when `isUpdated` flag is set to true cause I hadn't tested the case you've mentioned for the placeholder. Now I know. Cool. 27ccee7b7